server_admin/topics/realms/proc-creating-a-realm.adoc

```diff
@@ -13,7 +13,12 @@ realm and only be able to interact with customer-facing apps.
 
 . Point to the top of the left pane.
 
+ifeval::[{project_product}==true]
 . Click *Add Realm*.
+endif::[]
+ifeval::[{project_community}==true]
+. Click *Create Realm*.
+endif::[]
 +
 .Add realm menu
 image:{project_images}/add-realm-menu.png[Add realm menu]
```

Comment:
the file under Keycloak-images and should be renamed from create-realm-menu.png to add-realm-menu.png

Reply:
Also, the existing add-realm-menu.png under rhsso-images is working. You don't need a create-realm-menu.png there.

Reply:
Thanks @andymunro , this makes sense.